shlr/sdb/src/sdb_version.h

```diff
@@ -1 +0,0 @@
-#define SDB_VERSION "0.9.8"
```

Comment:
please do not include this file in the commit - looks like it is a glitch and should be in the gitignore or so, since it is autogenerated by the sdb sources when building.